counterparty-core/counterpartycore/server.py

```diff
@@ -303,6 +303,7 @@ def initialise_config(
 
     # Backend Poll Interval
     if backend_poll_interval:
+        assert backend_poll_interval >= 3.0
```

Comment:
@ouziel-slama can we specify the minimum value in the CLI `--help` and exit gracefully if the condition isn't met?

Reply:
sure!